app/views/feeds/_form_main.html.erb

```diff
@@ -0,0 +1,203 @@
+    <hr>
+    <h3><%= t(".headers.overrides") %></h3>
+    <p><%= t(".help.overrides") %></p>
+    <div class="mb-4">
+      <div class="form-floating">
+        <%= form.text_field :title %>
+        <%= form.label :title %>
+      </div>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_field :subtitle %>
+        <%= form.label :subtitle %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.subtitle") %>"><span class="material-icons">help</span></button>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_area :description %>
+        <%= form.label :description %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.description") %>"><span class="material-icons">help</span></button>
+    </div>
+
+    <div class="mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.text_area :summary %>
+        <%= form.label :summary %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.summary") %>"><span class="material-icons">help</span></button>
+    </div>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.private") %></h3>
+    <div class="prx-field-group">
+      <div>
+        <%= form.check_box :private %>
+        <%= form.label :private %>
+      </div>
+
+      <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.private") %>" data-bs-html="true"><span class="material-icons">help</span></button>
+    </div>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.auth_tokens") %></h3>
+    <p><%= t(".help.auth_tokens") %></p>
+    <%= render "form_auth_tokens", feed: feed, form: form %>
+  </div>
+
+  <div class="col-12">
+    <hr>
+    <h3><%= t(".headers.ad_zones") %></h3>
+    <p><%= t(".help.ad_zones") %></p>
+
+    <%= form.check_box :billboard, value: 0 %>
+    <%= form.label :billboard, class: "me-2" %>
+
+    <%= form.check_box :house, value: 0 %>
+    <%= form.label :house, class: "me-2" %>
+
+    <%= form.check_box :paid, value: 0 %>
+    <%= form.label :paid, class: "me-2" %>
+
+    <%= form.check_box :sonic_id, value: 0 %>
+    <%= form.label :sonic_id, class: "me-2" %>
+
+  </div>
+  <div class="col-12">
+    <hr>
+  </div>
+<% end %>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :url %>
+    <%= form.label :url %>
+  </div>
+
+  <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.url") %>"><span class="material-icons">help</span></button>
+</div>
+
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :new_feed_url %>
+    <%= form.label :new_feed_url %>
```

Comment:
New Feed Url is another field left off Feeder designs so not sure if i needed to bring it over.

Reply:
Yes, we do need it.